docs/source/release_notes.rst

```diff
@@ -4,18 +4,19 @@ Release Notes
     * Enhancements
     * Fixes
     * Changes
-        * Removed ``ComponentGraph.from_list`` and update PipelineBase implementation for creating pipelines from a list of components :pr:`2549`
+        * Updated ``PipelineBase`` implementation for creating pipelines from a list of components :pr:`2549`
```

Comment:
Updating release notes to make clear that this is actually the PR that removed `from_list` :)